### PR TITLE
Add credits to toolings

### PR DESCRIPTION
**Pull request by dem4ron**
Adds Credits component under tooling info sections.

### CODE REVIEW

File: app/views/tracks/build/show.html.haml [hunk 1]
```diff
@@ -200,7 +201,6 @@
                 = exercise.title
               .record-value #{exercise.num_completed} completions (30% of attempts)
 
-        = render ReactComponents::Track::Credits.new(Array.new(5, 'https://avatars.githubusercontent.com/u/66035744?v=4'), 1, "volunteer", 3, "hero")
```
Comment (ErikSchierboom) [inline app/views/tracks/build/show.html.haml:214]: Shouldn't this be replaced with a call to the renamed component?

Comment (dem4ron) [reply app/views/tracks/build/show.html.haml:214]: I've already replaced those. check above.

File: app/javascript/components/common/Credits.tsx [hunk 2]
```diff
@@ -23,12 +23,12 @@ export function Credits({
...
   const [overflow] = useState<number>(topCount + Number(bottomCount) - 2)
 
-  if (users.length === 0 || !users) {
+  if (!users || users.length === 0) {
     return null
   }
 
   return (
-    <div className={`flex items-center ${className}`}>
+    <div className={`flex items-center ${className ? className : ''}`}>
```
Comment (ErikSchierboom) [inline app/javascript/components/common/Credits.tsx:31]: Why is this change needed?

File: app/views/tracks/build/show.html.haml (hunk 1 above)
Comment (ErikSchierboom) [reply app/views/tracks/build/show.html.haml:214]: Ah

File: app/javascript/components/common/Credits.tsx (hunk 2 above)
Comment (dem4ron) [reply app/javascript/components/common/Credits.tsx:31]: in this case it fails 'quickerly', and doesn't error out with a `cannot read propertly 'length' of undefined`.

Comment (ErikSchierboom) [reply app/javascript/components/common/Credits.tsx:31]: I think that was fixed in https://github.com/exercism/website/pull/3159/files#diff-46fb86f1af44dd4f4a5165f8fba3c09d544a6651a560bffb1a9b8c0f5b63e3ecR26. Could you try reverting and see if it works?

Comment (ErikSchierboom) [reply app/javascript/components/common/Credits.tsx:31]:
```suggestion
    <div className={`flex items-center ${className}`}>
```
